src/main/java/org/jabref/model/database/BibDatabaseContext.java

```diff
@@ -211,4 +215,12 @@ public void convertToLocalDatabase() {
     public List<BibEntry> getEntries() {
         return database.getEntries();
     }
+
+    public Path getFulltextIndexPath() {
+        Path appData = Path.of(AppDirsFactory.getInstance().getUserDataDir(JabRefFrame.FRAME_TITLE, SearchFieldConstants.VERSION, "org.jabref"));
```

Comment:
JabRefFrame is totally a GUI thing, something we need to totally rework sometime in the future too. This is very bad in the model module. We need somehow a more generic solution here. @koppor

Reply:
Well, we could just use a literal 'JabRef'...

Reply:
No, no magic strings. Maybe olly got an idea.

Reply:
I checked the documentation at https://github.com/harawata/appdirs. Magic string `jabref` is OK for me here.